### DISCUSSION TITLE
Windows build using cmake fails

### DISCUSSION THREAD

**Discussion by danesh-d** [scivision/mumps-superbuild#65] [Q&A] [answered]
Hi all,

When I execute the command:

cmake -G Ninja

I get the following errors:

CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.

CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_Fortran_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!

Should set some environment variables? I have successfully installed Intel C/C++ and Fortran compilers and Intel MKL. Both C/C++ and Fortran compilers are callable since I have already executed the script that sets the env variables.

Regards,

D.

**Comment by scivision** [comment, marked as answer]
I think you fixed that now per other message. I updated Readme_windows.md to explain CMAKE_MAKE_PROGRAM.
